Add detailed information about DTI to model page (#310)
PennLINC/qsirecon: docs/models/dti.rst

Diffusion tensor imaging (DTI)
##############################

DTI Foundational Papers
=======================

**DTI Concept Introduction**: Basser et al. introduced DTI as a new MRI 
modality that computes a Gaussian diffusion tensor per voxel, yielding eigenvalues 
and eigenvectors that describe 3D water diffusion :cite:p:`basser1994`. 

> This enabled mapping of fiber orientations in tissue and orientation-independent 

Reviewer: ```suggestion
This enabled mapping of water diffusion orientations in tissue and orientation-independent 
```

scalar measures like trace (sum of eigenvalues, related to Mean Diffusivity, MD). 
It established the idea that tensors encode more microstructural information 
than single-direction diffusivities.

**First Anisotropy Indices**: Pierpaoli and Basser defined fractional anisotropy 
(FA), a normalized 0–1 index quantifying how anisotropic (directionally dependent)
diffusion is :cite:p:`pierpaolibasser1996`. 
They demonstrated that earlier methods (diffusion measured in just a few directions) 
underestimated anisotropy, and introduced rotationally invariant metrics (FA, relative 

> anisotropy, etc.) to robustly characterize white matter integrity. This work also 

Reviewer: ```suggestion
anisotropy, etc.). This work also 
```

cautioned that noise can bias anisotropy measures requiring eigenvalue ordering.

**First Human DTI Maps**: Pierpaoli et al. acquired the first DTI scans of the human 
brain, mapping principal diffusion directions and magnitudes 
:cite:p:`pierpaoli1996`. They observed that water diffuses ~3 times faster along 
axonal fibers than perpendicular to them in highly coherent tracts (e.g. corpus 

> callosum) than in regions with crossing or less organized fibers.

Reviewer 2: ```suggestion
callosum).
```
This sentence gets a bit long and convoluted. The 3 times faster is for one comparison (along fibers relative to perpendicular), so it's not clear what the word "than" means in this last fragment. I would suggest to remove the last fragment and add another sentence to discuss differences between regions, if needed.

They also introduced Trace (D = 3×MD) as 
an orientation-invariant measure of overall diffusivity, which was roughly 
uniform in normal brain except higher in cortical gray matter due to its 
higher water content.

**Eigenvalue-Derived Metrics**: By the early 2000s, researchers began interpreting 
individual tensor eigenvalues. Song et al. first showed that 
axial diffusivity (AD, diffusion along the primary eigenvector, λ1) and 
radial diffusivity (RD, diffusion perpendicular to axons, mean of λ2 & λ3) 
can provide pathologically specific insights :cite:p:`song2002`. 

> In a mouse model of demyelination, RD increased with myelin loss while AD stayed 

Reviewer 2: Citation needed? Or is this the Song study?

and slowly falls thereafter, while mean and radial diffusivities do the reverse 
(minimal in young adults, then rising in aging). Interestingly, this large study 
found no simple "last-in-first-out" pattern although late-maturing frontal tracts 
often showed pronounced aging changes, all regions eventually exhibited 
microstructural decline, indicating a widespread but heterogeneous aging effect 
rather than one specific sequence.

**Regional Patterns in Aging**: Many DTI studies of aging report that anterior white 
matter tracts (which myelinate last) are more vulnerable to aging. Salat et al. 
found significantly lower FA in older adults (mean age ~67) compared to young 
(mean ~24), especially in the frontal lobes and corpus callosum :cite:p:`salat2005`. 
In contrast, posterior tracts like the splenium of the callosum or occipital white 
matter showed smaller FA differences. Such findings support that age-related myelin 
degeneration and fiber loss are often greatest in late-developing, more complex pathways 
(though subsequent research has refined this view with more nuanced patterns).

**Microstructural Changes with Aging**: DTI metrics suggest that aging involves 

> loss of fiber integrity (lower FA) and increased water mobility 

Reviewer: ```suggestion
lower FA and increased water mobility 
```

(higher MD/apparent diffusion coefficient, ADC) :cite:p:`westlye2010`. 
In older adults, increased radial diffusivity is commonly observed, consistent 
with demyelination or degraded myelin packing, while axial diffusivity may also 
eventually decrease if axonal loss occurs. Longitudinal studies in elderly cohorts 
(e.g. over 60) have confirmed ongoing within-person FA declines annually. These DTI changes correlate with cognitive slowing and 
executive function decline in many studies, highlighting DTI's value in tracking 
brain aging and its cognitive consequences.

DTI Methodological Warnings and Caveats
=======================================

**Single Tensor Limitations (Crossing Fibers)**: The basic DTI model assumes one 
dominant fiber orientation per voxel – an assumption often violated in the brain. 
In regions with crossing, kissing, or branching fibers, the tensor model yields 
an average that can underestimate anisotropy and obscure fiber directions. For 
instance, a voxel containing two crossing tracts will show an artificially low 

> FA (appearing "isotropic") even if each tract is highly anisotropic. This issue 

Reviewer 2: ```suggestion
FA (appearing "isotropic") even if each tract is highly anisotropic, and principal diffusion direction that is an average of the true tract directions. This issue 
```

can lead to misinterpretation of reduced FA: it might reflect complex fiber 
geometry rather than neural degeneration. Advanced high-angular-resolution methods 
(e.g., multi-tensor or Q-ball imaging) are recommended when crossing fibers are 
prevalent, or one should interpret DTI metrics in such regions with caution 

> :cite:p:`wheelerkingshott2009`.

Reviewer: I'm not familiar with this one off the top of my head. Did it get picked because it is early?

FA appear higher in low-FA regions (background noise imposes a floor) 
:cite:p:`jones2010`. Head motion is another major concern: even subtle 
motion can cause directional-dependent blurring or signal drop-out, which may 
mimic or mask true diffusion anisotropy. For example, uncorrected motion can 
spuriously increase FA in gray matter or produce group differences unrelated to 
biology (as noted in studies of populations like children or patients who move 
more) :cite:p:`yendiki2014`. Best practices include using motion correction 
algorithms, excluding data with excessive motion, and using robust fitting methods 
(e.g., iteratively reweighted least squares) that down-weight outliers caused by 
artifacts.

**Acquisition and Analysis Choices**: The choice of diffusion gradient directions 
and b-value can influence DTI metrics. A minimal 6-direction tensor encoding is 
insufficient for reliable quantitative work – more directions (20–30+) are 
recommended to stabilize FA/MD measures and reduce variability. Similarly, 
moderate b-values (~1000 s/mm²) are typically chosen to balance SNR and 
sensitivity; very high b-values can introduce bias in tensor-fitting (and 

> require models beyond DTI). During analysis, image alignment (registration) 

Reviewer 2: ```suggestion
sensitivity; very high b-values can introduce bias in tensor-fitting, due to higher sensitivity to non-Gaussian diffusion -- and 
require other models, such as DKI. During analysis, image alignment (registration) 
```

and smoothing can also introduce caveats: misregistration across subjects can 
blur tract-specific values, and heavy smoothing can artificially increase FA 
in partial volume voxels. Tools like tract-based spatial statistics (TBSS) were 
developed to mitigate some of these issues by skeletonizing white matter maps 

> to focus on centers of tracts. The key caveat is that DTI analyses involve many 

Reviewer 2: I would drop the TBSS sentence.
